scripts/etc/init.d/rcS

```diff
@@ -760,7 +760,7 @@ cd /rootfs/boot/config/apt/
 # iterate through all the *.list files and add them to /etc/apt/sources.list.d
 for listfile in ./*.list
```

Comment:
I don't know what point you're trying to make here.
I just noticed that a test which was expected to return false actually returned true and I fixed that.
If you don't like the way that the file iterations are implemented now, code away and submit a PR with your changes and an explanation why it is better.